README.md

```diff
@@ -447,7 +446,7 @@ The scenario requiring use of this limit is straightforward, but note that hardl
 
 Consider a scenario where you have a single 6kW string inverter, and attached are two strings each of 5.5kW potential generation pointing in separate directions. This is considered "over-sized" from an inverter point of view. It is not possible to set an AC generation limit for Solcast that suits this scenario when configured as two sites, as in the mid-morning or afternoon in Summer a string may in fact be generating 5.5kW DC, with 5kW AC resulting, and the other string will probably be generating as well. So setting an AC limit in Solcast for each string to 3kW (half the inverter) does not make sense. Setting it to 6kW for each string also does not make sense, as Solcast will almost certainly over-state potential generation.
 
-The hard limit may be set in the integration configuration, or set via a service call in `Developer Tools`.
+The hard limit may be set in the integration configuration, or set via manually invoking the Action in `Developer Tools`.
```

Comment:
Prefer lower case action.  Yes, it might be capitalised in the, but unless highlighted (e.g. with backticks - not recommended), just refer to it as an action.

Reply:
done